### PR TITLE
Remove animations from the Popover menus

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

$ https://github.com/Expensify/App/issues/6121

### Tests
Tested All menus on the web.

### QA Steps
1. Go to staging.new.expensify.com and login.
2. Click the + icon in a Group DM text input and verify the popup menu.
3. Follow other menus as well.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web | Desktop

https://user-images.githubusercontent.com/24370807/142234892-06389ff9-7781-4afa-a6b9-dafba116a099.mp4

#### Mobile Web

https://user-images.githubusercontent.com/24370807/142239726-b9ac9dfc-2193-4097-a3bc-526385cdaebe.mp4

#### iOS

https://user-images.githubusercontent.com/24370807/142245110-060c22d9-6fd1-45c8-a0c9-79d812c00577.mp4

#### Android

https://user-images.githubusercontent.com/24370807/142238585-011fb836-23df-47c1-b3d2-e7496ddb78b5.mp4

### CODE REVIEW

**Comment by parasharrajat** [comment]
@Jag96 This is ready.

File: src/components/PopoverMenu/popoverMenuPropTypes.js [hunk 1]
```diff
@@ -46,12 +46,16 @@ const propTypes = {
 
     /** Optional non-interactive text to display as a header for any create menu */
     headerText: PropTypes.string,
+
+    /* Whether disable the animations */
```
Comment (Jag96) [inline src/components/PopoverMenu/popoverMenuPropTypes.js:50]: Update to `/**` per style guide

File: src/components/Popover/popoverPropTypes.js [hunk 2]
```diff
@@ -12,13 +12,17 @@ const propTypes = {
         bottom: PropTypes.number,
         left: PropTypes.number,
     }),
+
+    /* Whether disable the animations */
```
Comment (Jag96) [inline src/components/Popover/popoverPropTypes.js:16]: Same here, `/**`

File: src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js [hunk 3]
```diff
@@ -258,6 +258,7 @@ class PopoverReportActionContextMenu extends React.Component {
                     onModalHide={this.runAndResetOnPopoverHide}
                     anchorPosition={this.state.popoverAnchorPosition}
                     animationIn="fadeIn"
+                    disableAnimation={false}
```
Comment (Jag96) [inline src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js:261]: Why does this need to be set to false here?

**Review by Jag96 — CHANGES_REQUESTED** [review]
Looking good, just one style update and one question

File: src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js (hunk 3 above)
Comment (parasharrajat) [reply src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js:261]: I think we want to keep the animation for context menu. Let me otherwise.

Comment (Jag96) [reply src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js:261]: Ah for delete comment etc I see, I agree that is fine for the context menu

**Comment by parasharrajat** [comment]
@Jag96 changes done.

**Review by Jag96 — APPROVED** [review]

**Pull request merged**